File: CSharp/SpaceVIL/sandbox/View/JsonTest.cs
using System.IO;
using SpaceVIL;
using View.json;

namespace View
{
    class JsonTest : ActiveWindow
    {
        override public void InitWindow()
        {
            JsonApplier ja = new JsonApplier();
            var currentPath = Directory.GetCurrentDirectory() + Path.DirectorySeparatorChar;
            var jsWindow = "JsonLayout" + Path.DirectorySeparatorChar + "window.json";

            ja.ApplyJson(currentPath + jsWindow, this);

            var jsStyle = "JsonLayout" + Path.DirectorySeparatorChar + "style.json";
            ButtonCore btn = new ButtonCore("JSON");
            AddItem(btn);
            ja.ApplyJson(currentPath + jsStyle, btn);
        }
    }
}

File: Java/SpaceVIL/spacevil/sandbox/src/main/java/sandbox/View/JsonTest.java
package sandbox.View;

import com.spvessel.spacevil.*;
import com.spvessel.spacevil.Flags.ItemAlignment;
import com.spvessel.spacevil.Flags.SizePolicy;
import com.spvessel.spacevil.Frame;
import sandbox.View.json.JsonApplier;

import java.io.*;
import java.nio.file.*;
import java.lang.reflect.Field;
import java.lang.reflect.InvocationTargetException;
import java.lang.reflect.Method;
import java.util.List;

public class JsonTest extends ActiveWindow {
    @Override
    public void initWindow() {
        JsonApplier ja = new JsonApplier();
        String currentPath = Paths.get("").toAbsolutePath().toString() + File.separator;

        String jsWindow = "JsonLayout" + File.separator + "window.json";
        ja.applyJson(currentPath + jsWindow, this);

        String jsStyle = "JsonLayout" + File.separator + "style.json";
        ButtonCore btn = new ButtonCore("JSON");
        addItem(btn);
        ja.applyJson(currentPath + jsStyle, btn);
    }
}
